app.js

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env node
+const colors = require('colors');
+const path = require('path');
+const fs = require('fs');
+const markdownLinkExtractor = require('markdown-link-extractor');
+const fetch = require('node-fetch');
+
+let inputDoc = process.argv[2];
+let options = process.argv[3];
+
+const validateArgvExist = (inputFile) => {
+    if (inputFile === undefined) {
+        console.log('You need to enter a correct filename to continue'.brightRed);
+    } else validateExt(inputFile);
+};
+
+const validateExt = (inputFile) => {
+    let mdExt = path.extname(inputFile);
+        if (mdExt === '.md') {
+            readFiles(inputFile);
+        } else console.log('invalid format, only MD files are supported, try again'.brightRed); 
+};
+
+const readFiles = (inputFile) => {
+    fs.readFile(inputFile, 'utf8', (err, data) => {
+        if (err) {
+            console.log(err);
```

Comment:
Esto se pudo ver mejor con una promesa
```suggestion
            (res=>{
            then
            catch
```